Modules/CalcDefence.lua

```diff
@@ -1113,68 +1116,103 @@ function calcs.defence(env, actor)
 	end
 
 	--maximum hit taken
-	--FIX X TAKEN AS Y (output[damageType.."TotalPool"] should use the damage types that are converted to in output[damageType.."TakenHitMult"])
+	--FIX ARMOUR MITIGATION FOR THIS, uses input damage to calculate mitigation from armour, instead of maximum hit taken
 	for _, damageType in ipairs(dmgTypeList) do
-		output[damageType.."MaximumHitTaken"] = output[damageType.."TotalPool"] / output[damageType.."TakenHitMult"]
 		if breakdown then
-			breakdown[damageType.."MaximumHitTaken"] = {
-				s_format("Total Pool: %d", output[damageType.."TotalPool"]),
-				s_format("Damage Taken modifier: %.2f", output[damageType.."TakenHitMult"]),
-				s_format("Maximum hit you can take: %d", output[damageType.."MaximumHitTaken"]),
+			breakdown[damageType.."MaximumHitTaken"] = { 
+				label = "Maximum Hit Taken (uses lowest value)",
+				rowList = { },
+				colList = {
+					{ label = "Type", key = "type" },
+					{ label = "TotalPool", key = "pool" },
+					{ label = "Taken", key = "taken" },
+					{ label = "Final", key = "final" },
+				},
 			}
 		end
+		output[damageType.."MaximumHitTaken"] = math.huge
```

Comment:
Consider assigning `math.huge` to a local variable to speed up access like it is done with other members of the `math` and `table` modules which are called often.
Example:
```lua
local m_huge = math.huge
```